megamek/src/megamek/client/ui/swing/lobby/ChatLounge.java

```diff
@@ -1564,7 +1560,7 @@ private void createArmy() {
     }
 
     public void loadRandomSkills() {
-        clientgui.getRandomSkillDialog().showDialog(clientgui.getClient().getGame().getEntitiesVector());
+
```

Comment:
What's the point of having this method if it's empty?

Reply:
I forgot to delete it. Completely unused.